contracts/libraries/Token.sol

```diff
@@ -123,20 +117,7 @@ library Token {
       _success = tryTransferERC721(_token, _to, _info.id);
     } else revert ErrUnsupportedStandard();
 
-    if (!_success) {
-      revert(
-        string(
-          abi.encodePacked(
-            "Token: could not transfer ",
-            toString(_info),
-            " to ",
-            Strings.toHexString(uint160(_to), 20),
-            " token ",
-            Strings.toHexString(uint160(_token), 20)
-          )
-        )
-      );
-    }
+    if (!_success) revert ErrTokenCouldNotTransferFrom(_info, address(this), _to, _token);
```

Comment:
Should declare a new error `ErrTokenCouldNotTransfer` to distinguish from `ErrTokenCouldNotTransferFrom`.